src/main/java/seedu/cakecollate/model/OrderItemList.java

```diff
@@ -0,0 +1,105 @@
+package seedu.cakecollate.model;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.List;
+
+import javafx.collections.ObservableList;
+import seedu.cakecollate.model.orderitem.OrderItem;
+import seedu.cakecollate.model.orderitem.UniqueOrderItemList;
+
+public class OrderItemList implements ReadOnlyOrderItemList {
```

Comment:
If these two classes are supposed to be the equivalent of CakeCollate.java and ReadCakeCollate.java, maybe this could be better named after the model name? so something like public class OrderItem(s) implements ReadOnlyOrderItems. 

Something trivial, but I think it could make more sense while writing the guides, what do you think?

Reply:
oh yeah that's a good point, will do that